i18n/qqq.json

```diff
@@ -8,5 +8,12 @@
 	"wikibasereconcileedit-desc": "{{desc|name=WikibaseReconcileEdit}}",
 	"wikibasereconcileedit-editendpoint-invalid-reconcile-json": "Error returned by the API if the “reconcile” request parameter is invalid.",
 	"wikibasereconcileedit-editendpoint-unsupported-reconcile-version": "Error returned by the API if the “reconcile” request parameter specifies an unsupported version.\n\nParameters:\n* $1 – the supported versions (comma-separated list)\n* $2 – the number of supported versions",
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "Error returned by the API if the “reconcile” request parameter contains no valid field to specify the property ID to reconcile on.\n\nParameters:\n* $1 – the version of the request parameter, e.g. 0.0.1\n* $2 – the name of the field, e.g. urlReconcile"
+	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "Error returned by the API if the “reconcile” request parameter contains no valid field to specify the property ID to reconcile on.\n\nParameters:\n* $1 – the version of the request parameter, e.g. 0.0.1\n* $2 – the name of the field, e.g. urlReconcile",
+	"wikibasereconcileedit-editendpoint-invalid-type-property-must-be-url": "Error returned by the API if the “reconcile” request propertyId parameter is not of type URL.\n\nParameters:\n* $1 – the name of the property, e.g. P123\n* $2 – the name of the field, e.g. urlReconcile.\n* $3 – the datatype of the provided reconcile property, e.g. string",
+	"wikibasereconcileedit-editendpoint-unspecified-entity-input-version": "Error returned by the API if the “entity” request parameter contains no version field in the entity input",
+	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "Error returned by the API if the “entity” request parameter specifies an unsupported version.\n\nParameters:\n* $1 – the supported versions (comma-separated list)\n",
```

Comment:
The parameters here don’t look right – in the English message, $1 is the specified (unsupported) version, $2 the supported versions, and $3 the number of supported versions.

Reply:
You are right. Updated!